src/views/v2/tickets/item/extra.php

```diff
@@ -76,13 +76,14 @@
 	'tribe-tickets__tickets-item-extra--price-suffix' => $has_suffix,
 ];
 
+$context = $this->get_values();
 ?>
 <div <?php tribe_classes( $classes ); ?>>
 
-	<?php $this->template( 'v2/tickets/item/extra/price' ); ?>
+	<?php $this->template( 'v2/tickets/item/extra/price', $context ); ?>
 
-	<?php $this->template( 'v2/tickets/item/extra/available' ); ?>
+	<?php $this->template( 'v2/tickets/item/extra/available', $context ); ?>
 
-	<?php $this->template( 'v2/tickets/item/extra/description-toggle' ); ?>
+	<?php $this->template( 'v2/tickets/item/extra/description-toggle', $context ); ?>
```

Comment:
This shouldn't be necessary, if it is, we should fix it in a future sprint as part of a Common thing. This workaround shouldn't be needed/used.